src/file_pipe_log/pipe.rs

```diff
@@ -248,7 +246,7 @@ impl<F: FileSystem> SinglePipe<F> {
         let new_seq = writable_file.seq + 1;
         debug_assert!(new_seq > DEFAULT_FIRST_FILE_SEQ);
 
-        writable_file.writer.close()?;
+        writable_file.writer.close().unwrap();
```

Comment:
Add a comment to explain "Otherwise it will be considered a corruption at next restart".

Reply:
Can you explain more about this comment?

I guess "It" means the writeable file that is to be closed? then, who will consider it a corruption? and what does the "next restart" mean here, if it returns error instead of calling `unwrap()`, there won't be "restart"?

Reply:
I guess it's not needed anymore, after writer::sync can panic. The original case is when user calls engine::write and it fails, then he writes again, the file might be considered closed when physically it's not. Leaving a section of junk data and when user next restart the server, engine recovery will consider the junk data a potential corruption and refuse to continue.